test/functional/specs/Identity/C2581.js

```diff
@@ -31,7 +31,7 @@ test.meta({
 test("Test C2581: Queue requests until we receive an ECID.", async () => {
   const alloy = createAlloyProxy();
   await alloy.configure(debugEnabledConfig);
-  await alloy.sendEventAsync({ renderDecisions: true });
+  await alloy.sendEventAsync();
```

Comment:
For others looking at this, we thought that because `renderDecisions` was `true`, that it was causing rendering notifications to be sent to Konductor, which was causing `await t.expect(interactHook.numRequests).eql(2);` fail. I don't know why we had `{ renderDecisions: true }`.  That may have been the cause of the test failure, but, since `interactHook` is a SequentialHook, it may have been the problems with SequentialHook that @jonsnyder was/is investigating.